src/index.html

```diff
@@ -11,7 +11,184 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form id="myForm" action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <div class="group">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label for="surname" class="label-margin">
+            Surname:
+            <input 
+            type="text" 
+            id="surname" 
+            name="surname" 
+            size="20"
+            autocomplete="off"
+            >
```

Comment:
check spaces all across html file for all inputs
```suggestion
            <input 
              type="text" 
              id="surname" 
              name="surname" 
              size="20"
              autocomplete="off"
            >
```